pkg/apis/deployment/volumemount.go

```diff
@@ -161,11 +161,13 @@ func getCsiAzureVolumeMountName(componentName string, radixVolumeMount *radixv1.
 // GetCsiAzureVolumeMountType Gets the CSI Azure volume mount type
 func GetCsiAzureVolumeMountType(radixVolumeMount *radixv1.RadixVolumeMount) radixv1.MountType {
 	if radixVolumeMount.BlobFuse2 != nil {
-		switch radixVolumeMount.BlobFuse2.Protocol {
-		case radixv1.BlobFuse2ProtocolFuse2:
+		switch string(radixVolumeMount.BlobFuse2.Protocol) {
+		case string(radixv1.BlobFuse2ProtocolFuse2), "": // default protocol if not set
 			return radixv1.MountTypeBlobFuse2Fuse2CsiAzure
-		case radixv1.BlobFuse2ProtocolNfs:
+		case string(radixv1.BlobFuse2ProtocolNfs):
 			return radixv1.MountTypeBlobFuse2NfsCsiAzure
+		default:
+			return "unsupported"
 		}
```

Comment:
```suggestion
		switch radixVolumeMount.BlobFuse2.Protocol {
		case radixv1.BlobFuse2ProtocolFuse2, "": // default protocol if not set
			return radixv1.MountTypeBlobFuse2Fuse2CsiAzure
		case radixv1.BlobFuse2ProtocolNfs:
			return radixv1.MountTypeBlobFuse2NfsCsiAzure
		default:
			return "unsupported"
		}
```
Type conversion to string is not required